devops/ansible/install-dependencies-playbook.yml

```diff
@@ -0,0 +1,15 @@
+- hosts: 127.0.0.1
+  user: root
+  become: yes
+  become_method: sudo
```

Comment:
I haven't tested on windows, but would this be a problem running ansible-playbook on a windows machine? Don't bother too much if it needs workaround, running on linux and macos is sufficient I think.

Reply:
Ansible does not have support for windows. As a workaround, VM solutions could be used for windows.